static/js/fastsearch.js

```diff
@@ -70,7 +70,7 @@ document.getElementById("searchInput").onkeyup = function (e) {
 // apiKey: read-only public secret: https://docs.meilisearch.com/reference/features/authentication.html#key-types
 function loadMeiliSearch() {
     meilisearch = new MeiliSearch({
-        host: 'https://avdsearch.aquasec.com',
+        host: 'http://35.177.163.35',
```

Comment:
@itaysk there's an issue with this as making an HTTP request will get blocked by most browsers that don't allow cross-origin access (to http in this case). See an example below for the requests getting blocked by Firefox in this case.

<img width="1058" alt="image" src="https://github.com/user-attachments/assets/1df59a81-6794-43a1-9eb2-65198a7a61ae" />